### ISSUE TITLE
自己训练的precision

### ISSUE THREAD

**Issue by WenmuZhou** [cheerss/PixelLink-with-pytorch#3]
我用你的代码跑了下ICDAR 2015的数据集，
loss降到这么多时
![image](https://user-images.githubusercontent.com/12406017/44317466-05746880-a464-11e8-8f62-9028fb05ceab.png)
在训练集上的测试结果如下，
![image](https://user-images.githubusercontent.com/12406017/44317481-191fcf00-a464-11e8-9e90-287c83ab5b88.png)
这是因为我的还没收敛吗？我第一次训练文本检测的，不知道loss降到什么时候是收敛好了

**Comment by cheerss** [comment]
Actually the loss is small enough, the main reason why I said the project is still under developing is that the accuracy is not good enough.
I have compared my pytorch code with original tensorflow code carefully and modified accordingly. There is still very llllittle difference between them. According to the PixelLink's author, these little difference should not lead to too much precision loss, so I am still debug the code as well as suspect the difference between pytorch and tensorflow.

**Comment by WenmuZhou** [comment]
Ok, look forward to the next update

**Comment by ztoString** [comment]
@WenmuZhou 活捉老哥，麻烦问老哥一句，你感觉这个PixelLink和PSENet比较起来，哪个会强劲一点呢

**Comment by WenmuZhou** [comment]
@ztoString 无法解答啊，我还没跑过pixellink

**Comment by rsqai** [comment]
> @WenmuZhou 活捉老哥，麻烦问老哥一句，你感觉这个PixelLink和PSENet比较起来，哪个会强劲一点呢

活捉两个老哥，请问PSNet和PixelLink的对比做了吗？哪个强劲~

**Comment by ztoString** [comment]
这是来自QQ邮箱的自动回复邮件。
 
邮件已收到~
